src/Controller/BoxConfigurationController.php

```diff
@@ -34,6 +34,9 @@ public function index($id, BoxConfigurationRepository $boxConfigurationRepositor
             throw $this->createNotFoundException('Unknown box configuration');
         }
 
+        // HACK HACK HACK
```

Comment:
Maybe replace this with:
"Make sure languageCode is lowercase"

Reply:
No... this is an HACK and should be fixed in the serialisation process. Just don't known how.

Reply:
Okay... then maybe:
"Hack: Output the languageCode is lowercase"

Reply:
My idea was to talk with you or @turegjorup about the Symfony way to fix this. So will af better comment and @todo on it

Reply:
Fixed